src/common/components/discussion/index.tsx

```diff
@@ -66,23 +66,38 @@ import { ProfilePopover } from "../profile-popover";
 interface ItemBodyProps {
   entry: Entry;
   global: Global;
+  isRawContent: boolean;
 }
 
 export class ItemBody extends Component<ItemBodyProps> {
   shouldComponentUpdate(nextProps: Readonly<ItemBodyProps>): boolean {
     return this.props.entry.body !== nextProps.entry.body;
   }
 
+  componentDidUpdate(
+    prevProps: Readonly<ItemBodyProps>,
+    prevState: Readonly<{}>,
+    snapshot?: any
+  ): void {
+    // console.log("Componentdid update");
+    // console.log(this.props);
+  }
+
   render() {
-    const { entry, global } = this.props;
+    const { entry, global, isRawContent } = this.props;
 
     const renderedBody = { __html: renderPostBody(entry.body, false, global.canUseWebp) };
-
     return (
-      <div
-        className="item-body markdown-view mini-markdown"
-        dangerouslySetInnerHTML={renderedBody}
-      />
+      <>
+        {!this.props.isRawContent ? (
```

Comment:
Please, always check positive condition because negative one confusing

Reply:
> why this is closed, by accident perhaps @MTSaeed ?
 @feruzm I noticed that this PR had faulty commits which were done using force push so I closed this PR and created a new branch to remain out of trouble. I will create a new PR for this task and ask other developers for suggestions about the issue that we discussed in our last call.